source/_patterns/02-molecules/nav/demo/navs.twig

```diff
@@ -0,0 +1,111 @@
+{% embed '@atoms/grid/_grid-1-up.twig' with { container: true } %}
+    {% block column_1 %}
+      <h1>Navs</h1>
+      {% include '@molecules/card/_card-helper.twig' with {
+        card_text: navs_helper,
+        card_background: 'secondary',
+        text_color: 'light',
+        button: {
+          button_link: 'http://getbootstrap.com/docs/4.0/components/navs/',
```

Comment:
In `navs` and `cards` we should be providing examples of using buttons with multiple elements. Having the method of override contained in the component is a nice "saved you a click" for people that'll be using this.